megamek/src/megamek/common/Compute.java

```diff
@@ -7391,8 +7388,9 @@ public static boolean useSpheroidAtmosphere(Game game, Entity en) {
             return false;
         }
         // aerodyne's will operate like spheroids in vacuum
+        PlanetaryConditions conditions = game.getPlanetaryConditions();
         if (!((IAero) en).isSpheroid()
-                && !game.getPlanetaryConditions().isVacuum()) {
+                && !conditions.getAtmosphere().isLighterThan(Atmosphere.THIN)) {
```

Comment:
Trace is lighter than thin. Is this intended?

Reply:
The previous function had a confusing name 

    public boolean isVacuum() {
        return (atmosphere == ATMO_VACUUM) || (atmosphere == ATMO_TRACE);
    }